### DISCUSSION TITLE
Google Play Data Safety section

### DISCUSSION THREAD

**Discussion by codyrotwein** [fastlane/fastlane#19851] [Q&A]
As you are probably aware, Google Play will start displaying a data safety section on the store starting next month, and enforcing developers to populate the section in April: https://android-developers.googleblog.com/2021/10/launching-data-safety-in-play-console.html 

They have given a way to export/import a CSV to make it easier. Are there any plans for fastlane to support this, or is it not part of the google play api? Can anyone lead me in the right direction as how I could possibly automate doing this? We have over 300 apps that would need get updated. Thanks!

**Comment by pkwiatkowski0** [comment]
We also have hundreds of apps and it would be a real pain for us to update privacy details for each one of them. Do you plan to add this functionality in fastlane?
Best regards

**Reply by gholias** [reply]
I'm also hoping for a solution in Fastlane to be easier to upload the CSV file.

**Comment by vaibhavantil1** [comment]
@codyrotwein - Jumping in here. While I don't have a solution for automating this in Fastlane, you can still automate generating the data safety csv with my free tool here: https://github.com/Privado-Inc/privado

Let me know if you have any questions/feedback.

**Comment by pkwiatkowski0** [comment]
In fact they postponed the deadline to July 20, but we still got no reply from Fastlane. Any updates on this?

**Comment by ppamorim** [comment]
Any update on this? I will try the Privado tool right now but we would like to have fastlane doing that.

**Comment by vaibhavantil2** [comment]
@ppamorim - Unfortunately there are no Play Store APIs to do that.

**Comment by chrisvire** [comment]
Does anyone know where to submit feature requests to Google's Play Store API? Are fastlane developers in regular contact with Google's Play Store API developers?

**Comment by owenbean400** [comment]
There is a Google developer API to upload data safety form. Here is a Fastlane plugin I developed to upload csv file to Google Play Console. https://github.com/owenbean400/fastlane-plugin-google_data_safety.

**Reply by france-workmail** [reply]
Thanks for this, do you happen to have something similar for health and/or financial features declaration as well?

**Reply by owenbean400** [reply]
@france-workmail I do not see any documentation for health nor financial features from Google. Furthermore, there are several items in app content section that does not to appear to have API access to update.
